plugins/receivers/slackchannel/config.go

```diff
@@ -0,0 +1,59 @@
+package slackchannel
+
+import (
+	"fmt"
+
+	"github.com/goto/siren/plugins/receivers/slack"
+)
+
+// ReceiverConfig is a stored config for a slack receiver
+type ReceiverConfig struct {
+	SlackReceiverConfig slack.ReceiverConfig `mapstructure:",squash"`
+	ChannelName         string               `json:"channel_name" mapstructure:"channel_name"`
+	ChannelType         string               `json:"channel_type" mapstructure:"channel_type"`
```

Comment:
I might've missed something, but what is `ChannelType` for?

Reply:
It is possible to send notification to either `user` or `channel`. This can be left empty and by default we send notification to `channel`.